plugins/modules/cloud/misc/proxmox_kvm.py

```diff
@@ -1233,7 +1246,7 @@ def main():
                 module.exit_json(changed=False, vmid=vmid, msg="VM with vmid <%s> already exists" % vmid)
             elif proxmox.get_vmid(name, ignore_missing=True) and not (update or clone):
                 module.exit_json(changed=False, vmid=proxmox.get_vmid(name), msg="VM with name <%s> already exists" % name)
-            elif not (node, name):
+            elif (not node) or (not name):
```

Comment:
The original condition `not (node, name)` is always `False`. Changing this condition is a change that must be documented in the changelog. Also this is likely a bugfix and should not be part of a feature PR so it can be backported.